### ISSUE TITLE
Some sort of API testing extension (like postman)

### ISSUE THREAD

**Issue by jawadcode** [OnlyBens/vscode-extension-ideas#45]
So one does not have to open postman every goddamn time just to test even simple APIs

**Comment by sereneinserenade** [comment]
https://marketplace.visualstudio.com/items?itemName=humao.rest-client

not exactly like postman but is something

**Comment by samarmohan** [comment]
@sereneinserenade, yes we know about that but I think @jawadcode is trying to say that we need a better GUI but INSIDE VSCode.

**Comment by sereneinserenade** [comment]
> @sereneinserenade, yes we know about that but I think @jawadcode is trying to say that we need a better GUI but INSIDE VSCode.

of course, that's why the words `not exactly like postman but is something` were there in my comment @samarmohan

**Comment by samarmohan** [comment]
lol ok
